app/models/plant.rb

```diff
@@ -0,0 +1,8 @@
+class Plant < ApplicationRecord
+  enum health_status: [:dead, :wither, :bloom, :normal]
+
+  validates :health_status, inclusion: { in: health_statuses.keys }
```

Comment:
Did you test this validation? Are you 100% sure it works? 🙂

Reply:
Yup